src/sources/kubernetes_logs/k8s_paths_provider.rs

```diff
@@ -29,10 +30,14 @@ impl K8sPathsProvider {
     }
 }
 
+#[derive(Debug, Snafu)]
+pub enum K8sPathsProviderError {}
```

Comment:
I like this empty enum, it's essentially an infallible equivalent, right? Although, it might be more explicit to just use `std::convert::Infallible`

Reply:
TIL about that one!